mRemoteV1/mRemoteV1.csproj

```diff
@@ -59,23 +59,40 @@
     <Reference Include="Geckofx-Winforms, Version=45.0.32.0, Culture=neutral, PublicKeyToken=3209ac31600d1857, processorArchitecture=x86">
       <HintPath>..\packages\Geckofx45.45.0.32\lib\net45\Geckofx-Winforms.dll</HintPath>
     </Reference>
+    <Reference Include="Google.Protobuf, Version=3.5.1.0, Culture=neutral, PublicKeyToken=a7d26565bac4d604, processorArchitecture=MSIL">
+      <HintPath>..\packages\Google.Protobuf.3.5.1\lib\net45\Google.Protobuf.dll</HintPath>
+    </Reference>
     <Reference Include="log4net, Version=2.0.8.0, Culture=neutral, PublicKeyToken=669e0ddf0bb1aa2a, processorArchitecture=MSIL">
       <HintPath>..\packages\log4net.2.0.8\lib\net45-full\log4net.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.CSharp" />
     <Reference Include="mscorlib" />
+    <Reference Include="MagicLibrary, Version=1.7.4.0, Culture=neutral, PublicKeyToken=3a6eb82f876a49bc">

```

Comment:
MagicLibrary has been removed from mRemoteNG - this was just done recently: see commit 8281f488f3c32e84d10d9567c90620388d43007b

So the MagicLibrary reference needs to be removed. Looks like there were very minimal UI changes (just in the settings) and I don't believe that area used MagicLib at all - so this should be a quick removal with no real ramifications in theory.

Reply:
Yeah, I don't purport to know VS projects very well at all - I suspect there will be more changes to the `.csproj` that are incorrect.

Reply:
Done - pushed a rebase to include this in the main commit